client/src/components/avatar/avatar.jsx

```diff
@@ -1,17 +1,23 @@
 import s from './avatar.module.sass';
 
 
-const Avatar = ({url, handlerClick, to, style}) =>{
+const Avatar = ({url, size}) =>{
+
+    let styleAvatar = null;
+    switch (size){
+        case 'small': styleAvatar = s.small;
+            break;
+        case 'medium': styleAvatar = s.medium;
+            break;
+        case 'large': styleAvatar = s.large;
+            break;
+        default: styleAvatar = s.small;
+    }
```

Comment:
this also can be solved with `classnames` library

check this out: https://www.npmjs.com/package/classnames